scripts/azureml-assets/azureml/assets/release/e2e_test.py

```diff
@@ -17,11 +17,17 @@
     parser.add_argument("-s", "--subscription", required=True, type=str, help="Subscription ID")
     parser.add_argument("-r", "--resource-group", required=True, type=str, help="Resource group name")
     parser.add_argument("-w", "--workspace-name", required=True, type=str, help="Workspace name")
+    parser.add_argument("-c", "--coverage-report", required=False, type=Path, help="Path of coverage report yaml")
```

Comment:
what's a sample coverage report?

Reply:
warning: Not all assets in next stage create list are covered by completed test jobs. Uncovered assets: ['microsoft.azureml.automl.forecasting.component', 'microsoft_azureml_automl_image_classification_component', 'microsoft_azureml_rai_tabular_explanation', 'microsoft.azureml.automl.image.classification.component', 'microsoft.azureml.automl.image.object.detection.component', 'rai_insights_explanation', 'microsoft_azureml_automl_text_classification_component', 'train_instance_segmentation_model', 'microsoft_azureml_automl_image_classification_multilabel_component', 'microsoft.azureml.automl.text.classification.component', 'microsoft.azureml.automl.text.classification.multilabel.component', 'microsoft_azureml_automl_text_classification_multilabel_component', 'microsoft.azureml.automl.regression.component', 'microsoft_azureml_automl_text_ner_component', 'microsoft_azureml_automl_image_object_detection_component', 'rai_insights_constructor', 'microsoft_azureml_rai_tabular_insight_gather', 'microsoft_azureml_automl_forecasting_component', 'microsoft_azureml_rai_tabular_insight_constructor', 'microsoft.azureml.automl.image.instance.segmentation.component', 'microsoft.azureml.automl.image.classification.multilabel.component', 'microsoft_azureml_automl_image_instance_segmentation_component', 'microsoft_azureml_automl_classification_component', 'microsoft_azureml_rai_tabular_counterfactual', 'rai_insights_causal', 'microsoft_azureml_rai_tabular_erroranalysis', 'microsoft_azureml_automl_regression_component', 'rai_insights_erroranalysis', 'microsoft_azureml_rai_tabular_score_card', 'microsoft.azureml.automl.classification.component', 'microsoft.azureml.automl.text.ner.component', 'train_object_detection_model', 'microsoft_azureml_rai_tabular_causal', 'rai_insights_counterfactual', 'rai_insights_gather', 'rai_score_card', 'train_image_classification_model', 'tensorflow-2.8-ubuntu20.04-py38-cuda11-gpu', 'tensorflow-2.9-ubuntu20.04-py38-cuda11-gpu', 'pytorch-1.11-ubuntu20.04-py38-cuda11-gpu', 'sklearn-1.1-ubuntu20.04-py38-cpu'].

Reply:
Oh I think I should exclude `environment` type

Reply:
how about test failure? How does the report look like?

Reply:
Test failure will just be reflected by uncovered assets. The group_test.py will have brief test results to indicate which tests failed or not.